src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IIsPatternExpression.cs

```diff
@@ -797,7 +855,42 @@ void M(int? x1, int x2, bool b)
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedFlowGraph, expectedDiagnostics);
         }
 
-        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Patterns)]
+        [Fact]
+        public void IsPattern_RecursivePattern()
```

Comment:
It doesn't look like we have a good coverage of scenarios when different pieces of a recursive pattern are present/missing/empty/error cases

Reply:
> It doesn't look like we have a good coverage of scenarios when different pieces of a recursive pattern are present/missing/empty/error cases 

This thread is marked as resolved, but I didn't see any new tests added 

---
In reply to: [244214596](https://github.com/dotnet/roslyn/pull/32018#discussion_r244214596) [](ancestors = 244214596)

Reply:
Already noted in [https://github.com/dotnet/roslyn/issues/32012](https://github.com/dotnet/roslyn/issues/32012)

---
In reply to: [244427900](https://github.com/dotnet/roslyn/pull/32018#discussion_r244427900) [](ancestors = 244427900,244214596)